README.md

```diff
@@ -95,7 +95,11 @@ Run tests with:
 # within project directory
 ./run_tests
 ```
+### Unattended Install
+To do an unattended install (if you already have a config)
 
+`~/.tmux/plugins/tpm/scripts/install_plugins.sh`
+fixes #277 and #193
```

Comment:
I wasn't explicit enough. Can you provide a description of these issues, but link to them. I was thinking something like:

```markdown
### Unattended Install

To do an unattended install, for example if you already have a config:

`~/.tmux/plugins/tpm/scripts/install_plugins.sh`

Note this is also a good workaround if you have other issues with the automatic installation, such as known issues with MacOS (see [#277](https://github.com/tmux-plugins/tpm/issues/277) and [#193](https://github.com/tmux-plugins/tpm/issues/277))
```

Note too by the way that I don't have merge permissions so this may not be merged, maintenance seems low in this repo judging by recent activity.